jetty-server/src/main/java/org/eclipse/jetty/server/HttpInput.java

```diff
@@ -535,37 +546,23 @@ protected void skip(Content content, int length)
      * @throws IOException if the wait is interrupted
      */
     protected void blockForContent() throws IOException
-    {
+    {   
         try
         {
-            _waitingForContent = true;
-            _channelState.getHttpChannel().onBlockWaitForContent();
-
-            boolean loop = false;
-            long timeout = 0;
-            while (true)
+            if (!_waitingForContent)
```

Comment:
@sbordet I have been testing just this change, which is for spurious wakeups.   It makes the following test fail for me about 50% of the time: `org.eclipse.jetty.client.ssl.SslBytesServerTest.testRequestWithBigContentWithSplitBoundary()`

So we need to reconsider how spurious wakeups should be handled here